onnxmltools/convert/common/_container.py

```diff
@@ -112,8 +113,9 @@ class ModelComponentContainer:
     encapsulated in a ONNX ModelProto.
     '''
 
-    def __init__(self, targeted_onnx):
+    def __init__(self, float_precision=32, targeted_onnx):
         '''
+        :param float_precision: precision of float number, 32 bits or 16 bits
```

Comment:
Precision of floats (or Precision of floating-point numbers), 32 bits or 16 bits. Applicable to ONNX inputs, outputs and initializers.